buffer/outputs.tf

```diff
@@ -54,3 +54,9 @@ output "cloudsql_app_stairway_db_creds" {
   description = "Terra Resource Buffer Service Stairway database user credentials"
   sensitive   = true
 }
+
+output "pool_name_to_folder_id" {
+  value       = var.enable ? "${zipmap(var.pool_list, google_folder.buffer_per_pool_folders)}" : {}
+  description = "Map from pool name to the folder that will contain all projects created for the pool"
+  sensitive   = true
```

Comment:
I'm no expert, but I don't think this is sensitive. This is also under the ### cloudsql outputs header. mind doing some header modification?

Reply:
Done.